eng/pipelines/templates/variables/globals.yml

```diff
@@ -4,4 +4,4 @@ variables:
   MemoryOptions: '-Xmx3072m'
   #Agent.Source.Git.ShallowFetchDepth: 1
   skipComponentGovernanceDetection: true  
-  EmulatorMsiUrl: 'https://acpedaily1.blob.core.windows.net/emulator/azure-cosmos-emulator.msi'
+  EmulatorMsiUrl: 'https://aka.ms/cosmosdb-emulator'
```

Comment:
For my information what is the primary difference here? Are we switching from the daily to a official public version of the emulator? I guess the main question is what emulator is published at `https://aka.ms/cosmosdb-emulator`.

Reply:
https://aka.ms/cosmosdb-emulator points to the public emulator and it is the emulator against which we have been gating changes for other languages (JS and Python).